docs/proxy/datacenter_proxy/examples.md

```diff
@@ -0,0 +1,700 @@
+        agent: {
+            http: new HttpProxyAgent(proxyUrl),
+            https: new HttpsProxyAgent(proxyUrl)
+        }
+    });
+    console.log(response.body)
+};
+
+useProxy();
+// Should return the same clientIp as
+useProxy();
+</marked-tab>
+
+
+<marked-tab header="Python 3" lang="python">
+import urllib.request as request
+import ssl
+
+def do_request():
+    # Replace <YOUR_PROXY_PASSWORD> below with your password
+    # found at https://my.apify.com/proxy
+    password = "<YOUR_PROXY_PASSWORD>"
+    proxy_url = f"http://session-my_session:{password}@proxy.apify.com:8000"
+    proxy_handler = request.ProxyHandler({
+        "http": proxy_url,
+        "https": proxy_url,
+    })
+
+    ctx = ssl.create_default_context()
+    ctx.check_hostname = False
+    ctx.verify_mode = ssl.CERT_NONE
+    httpHandler = request.HTTPSHandler(context=ctx)
+
+    opener = request.build_opener(httpHandler,proxy_handler)
+    return opener.open("https://api.apify.com/v2/browser-info").read()
+
+print(do_request())
+print("Should return the same clientIp as ")
+print(do_request())
+</marked-tab>
+
+
+<marked-tab header="Python 2" lang="python">
+import six
+from six.moves.urllib import request
+import ssl
+
+def do_request():
+    # Replace <YOUR_PROXY_PASSWORD> below with your password
+    # found at https://my.apify.com/proxy
+    password = "<YOUR_PROXY_PASSWORD>"
+    proxy_url = (
+        "http://session-my_session:%s@proxy.apify.com:8000" %
+        (password)
+    )
+    proxy_handler = request.ProxyHandler({
+        "http": proxy_url,
+        "https": proxy_url,
+    })
+
+    ctx = ssl.create_default_context()
+    ctx.check_hostname = False
+    ctx.verify_mode = ssl.CERT_NONE
+    httpHandler = request.HTTPSHandler(context=ctx)
+
+    opener = request.build_opener(httpHandler,proxy_handler)
+    return opener.open("https://api.apify.com/v2/browser-info").read()
+
+print(do_request())
+print("Should return the same clientIp as ")
+print(do_request())
+</marked-tab>
+
+
+<marked-tab header="PHP" lang="php">
+<?php
+function doRequest() {
+    $curl = curl_init("https://api.apify.com/v2/browser-info");
+    curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
+    curl_setopt($curl, CURLOPT_PROXY, "http://proxy.apify.com:8000");
+    // Replace <YOUR_PROXY_PASSWORD> below with your password
+    // found at https://my.apify.com/proxy
+    curl_setopt($curl, CURLOPT_PROXYUSERPWD, "session-my_session:<YOUR_PROXY_PASSWORD>");
+    $response = curl_exec($curl);
+    curl_close($curl);
+    return $response;
+}
+$response1 = doRequest();
+$response2 = doRequest();
+echo $response1;
+echo "\nShould be contain same clientIp as\n";
+echo $response2;
+?>
+</marked-tab>
+```
+
+### [](#two-requests-with-different-ip-addresses) Two requests with different IP addresses
+
+The IP addresses are chosen from the `SHADER` and `BUYPROXIES94952` proxy groups, however you don't have to specify proxy groups to use different IPs.
```

Comment:
I still don't get this too much. Why conflate proxy groups and IP rotation into the same example? Maybe that's only my opinion, but I always thought that examples should provide the minimal workable solution to the problem at hand, without any extra features or unneeded clutter. If we want to show off proxy groups, we should add proxy group examples.

Reply:
Yeah, that's a good point, I'm going to add separate examples for using proxy groups

Reply:
Actually, by the looks of it, the 'multiple requests with different IPs' example looks the same as 'single request with a random IP' one. The only difference is that the `useProxy()` function is called twice. 

So, actually this can be refactored to be a lot more economical